### ISSUE TITLE
Add category link to Post Grid block

### ISSUE THREAD

**Issue by mikemcalister** [studiopress/atomic-blocks#155]
We should output a category link in the post grid display, as it's usually seen along with posts like this.

**Comment by mindctrl** [comment]
This probably needs to be optional and off by default otherwise it will change the display on every site using it now.

**Comment by mikemcalister** [comment]
Yup, agreed. We'll add it as a toggle that is off by default.

**Comment by EricJammin** [comment]
Any updates on this? Or at least a code snippet we could add to achieve this in the meantime on our own sites?

**Comment by mikemcalister** [comment]
@EricJammin Yup, we have a few folks looking at this issue now. Should be available in the next couple updates. Thanks for your patience!

**Comment by pierluigiferrara** [comment]
This should be a useful option, is it still under development?

**Comment by mindctrl** [comment]
@pierluigiferrara it turned into a little rabbit hole and we put it on the back burner for now. It's definitely still on our radar, but we don't have an ETA yet.

**Comment by christinaleans** [comment]
Did anybody come up with a way to do this? I've been creating a site for a few weeks, almost done and the owner just decided the categories need to show. If I can't figure this out, I'm going to have to start over. :(

**Comment by yakabovski** [comment]
Is this still on the radar? Would be super useful to pull this into the posts grid.

**Comment by tracikay** [comment]
Just adding another note here that this would be very useful. Last update was in Jan. Is it still in development?

**Comment by jordonrupp** [comment]
Agreed +1 👍

**Comment by cdils** [comment]
+1 to this feature

**Comment by mikemcalister** [comment]
Hey folks! Yup, the category link is still on our radar and we're hoping to swing back around to the post grid block soon!
